console/services/service_services.py

```diff
@@ -250,7 +250,8 @@ def get_build_infos(self, tenant, service_ids):
                             bean["market_error_code"] = e.error_code
                             bean["version"] = service_source.version
                             bean["app_version"] = service_source.version
-                            return bean
+                            build_infos[service.service_id] = bean
+                            return build_infos
```

Comment:
Change the data structure?

Reply:
Yes, in order to fix the slow problem of component copying to obtain the build source, the method was changed to obtain multiple build sources, which is omitted here